src/module.ts

```diff
@@ -155,18 +154,32 @@ export default defineNuxtModule<ModuleOptions>({
     const { resolve } = createResolver(import.meta.url)
 
     // 4. Add the correct nuxt-auth app composable, for the desired backend
-    addImports([
-      {
-        name: 'useAuth',
-        from: resolve(`./runtime/composables/${options.provider.type}/useAuth`)
-      },
-      {
-        name: 'useAuthState',
-        from: resolve(
-          `./runtime/composables/${options.provider.type}/useAuthState`
-        )
-      }
-    ])
+    if (options.isEnabled) {
+      addImports([
+        {
+          name: 'useAuth',
+          from: resolve(`./runtime/composables/${options.provider.type}/useAuth`)
+        },
+        {
+          name: 'useAuthState',
+          from: resolve(
+            `./runtime/composables/${options.provider.type}/useAuthState`
+          )
+        }
+      ])
+    } else {
+      // In case nuxt-auth is disabled, we add stubs for the imports
+      addImports([
+        {
+          name: 'useAuth',
+          from: resolve('./runtime/composables/stub/useAuth')
+        },
+        {
+          name: 'useAuthState',
+          from: resolve('./runtime/composables/stub/useAuthState')
+        }
+      ])
+    }
```

Comment:
I believe you need to do a
```ts
if (!options.isEnabled) {
  return
}
```

before step 6?

Otherwise you end up adding runtime plugins and middleware, is this intended?